foreach1.js

```diff
@@ -9,3 +9,11 @@
 var foods = ["pizza", "tacos", "ice cream", "sushi"];
 
 // your code here
+
+for(let i = 0; i < foods.length; i++) {
+    console.log("I like " + foods[i])
+  }
+  
+  foods.forEach(function(food) {
+  //foodLove.push(food);
+  })
```

Comment:
the point of these iterators is to not use loops...

for this problem, you started off in line 17 exactly right! the missing part was the inside of the callback. if you think of the ONE `food` you have in the callback, what you want to do is `console.log()` "I like <food>." you can take line 14 from the `for` loop and put it in line 18 (replacing `food` for `foods[i]`), and it would work perfectly:

  foods.forEach(function(food) {
    console.log("I like " + food)
  })

Let me know if you still have questions